src/transformStateWithClones.js

```diff
@@ -7,7 +7,31 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let clone = Object.assign({}, state);
```

Comment:
Issue related to checklist item 5: Use a more descriptive name for the object copy, such as `stateCopy` instead of `clone`.